_episodes_rmd/11-writing-data.Rmd

```diff
@@ -71,7 +71,7 @@ Open up this document and have a look.
 > > ```{r, eval = FALSE}
 > > pdf("Life_Exp_vs_time.pdf", width = 12, height = 4)
 > > 
-> > ggplot(data = gapminder, aes(x = year, y = lifeExp, colour = country)) + 
+> > p = ggplot(data = gapminder, aes(x = year, y = lifeExp, colour = country)) + 
```

Comment:
We try to use the assignment operator `<-` instead of `=` outside of parameter setting. Can you change it here?